src/main/java/org/springframework/data/repository/query/ExtensionAwareQueryMethodEvaluationContextProvider.java

```diff
@@ -66,55 +81,38 @@ public ExtensionAwareQueryMethodEvaluationContextProvider(List<? extends Evaluat
 
 		Assert.notNull(extensions, "EvaluationContextExtensions must not be null");
 
-		this.delegate = new org.springframework.data.spel.ExtensionAwareEvaluationContextProvider(extensions);
+		this.delegate = new QueryMethodValueEvaluationContextAccessor(null, extensions);
```

Comment:
why are we using `null` here for the `Environment`? In L:71 a new `StandardEnvironment` is passed on.